packages/components/psammead-image/src/index.test.jsx

```diff
@@ -1,10 +1,29 @@
-import Image, { Img } from '.';
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
 import snapshotTests from './testHelpers/snapshotTests';
+import { landscape } from './testHelpers/fixtureData';
+import Image, { Img } from '.';
 
-describe("Image - imported as '{ Img }'", () => {
+describe("Image - imported as default 'Image'", () => {
   snapshotTests(Img);
 });
 
-describe("Image - imported as default 'Image'", () => {
+describe("Image - imported as '{ Img }'", () => {
   snapshotTests(Image);
 });
+
+describe("Image - imported as default 'Image'", () => {
```

Comment:
These two can probably be nested inside the describes in lines 7 and 11, as they appear to have name duplicates :)